examples/overlapping_circles.wgsl

```diff
@@ -21,7 +21,7 @@ fn circle(c: vec2<f32>, r: f32, probe: vec2<f32>) -> bool {
     }
 }
 
-let colors: array<vec3<f32>, 10> = array<vec3<f32>, 10>(
+var colors: array<vec3<f32>, 10> = array<vec3<f32>, 10>(
```

Comment:
Is this necessary for correct behavior? (I ask because I previously encountered an issue in naga where changing a let to var fixed it; if that's again the case I'd like to report it upstream)

https://github.com/gfx-rs/naga/issues/910